pkg/kubectl/describe.go

```diff
@@ -1882,6 +1886,70 @@ func (p *PetSetDescriber) Describe(namespace, name string, describerSettings Des
 	})
 }
 
+type CertificateSigningRequestDescriber struct {
+	client *client.Client
+}
+
+func (p *CertificateSigningRequestDescriber) Describe(namespace, name string, describerSettings DescriberSettings) (string, error) {
+	csr, err := p.client.Certificates().CertificateSigningRequests().Get(name)
+	if err != nil {
+		return "", err
+	}
+
+	cr, err := utilcertificates.ParseCertificateRequestObject(csr)
+	if err != nil {
+		return "", fmt.Errorf("Error parsing CSR: %v", err)
+	}
+
```

Comment:
do we want an indication of the approval status (like in the printer), and whether or not a cert has been issued (nil check on status.cert)?